Lombiq.Privacy.Samples/Recipes/Lombiq.Privacy.Samples.Content.recipe.json

```diff
@@ -0,0 +1,504 @@
+                      "DisplayText": "",
+                      "Latest": false,
+                      "Published": false,
+                      "ModifiedUtc": "2022-05-12T19:00:59.8117192Z",
+                      "PublishedUtc": null,
+                      "CreatedUtc": null,
+                      "Owner": "",
+                      "Author": "admin",
+                      "PrivacyConsentCheckboxPart": {
+                        "ConsentCheckbox": false,
+                        "ShowAlways": true
+                      },
+                      "FlowMetadata": {
+                        "Alignment": 3,
+                        "Size": 100
+                      }
+                    },
+                    {
+                      "ContentItemId": "4qn4t1wngqc5j4vvggk1ngae5e",
+                      "ContentItemVersionId": null,
+                      "ContentType": "Button",
+                      "DisplayText": null,
+                      "Latest": false,
+                      "Published": false,
+                      "ModifiedUtc": "2022-05-12T19:00:59.8155113Z",
+                      "PublishedUtc": null,
+                      "CreatedUtc": null,
+                      "Owner": "",
+                      "Author": "admin",
+                      "FormInputElementPart": {
+                        "Name": null
+                      },
+                      "FormElementPart": {
+                        "Id": null
+                      },
+                      "ButtonPart": {
+                        "Text": "Register",
+                        "Type": "submit"
+                      },
+                      "FlowMetadata": {
+                        "Alignment": 3,
+                        "Size": 100
+                      }
+                    }
+                  ]
+                },
+                "FlowMetadata": {
+                  "Alignment": 3,
+                  "Size": 100
+                }
+              }
+            ]
+          }
+        }
+      ]
+
+    },
+    {
+      "name": "Settings",
+      "RegistrationSettings": {
+        "UsersCanRegister": 1,
+        "UsersMustValidateEmail": false,
+        "UsersAreModerated": false,
+        "UseSiteTheme": false,
+        "NoPasswordForExternalUsers": false,
+        "NoUsernameForExternalUsers": false,
+        "NoEmailForExternalUsers": false,
+        "UseScriptToGenerateUsername": false,
+        "GenerateUsernameScript": "/* Uncomment to map AzureAD\r\n// Uncomment to output the context variable in the logs\r\n// log(\"warning\", JSON.stringify(context));\r\nswitch (context.loginProvider) {\r\n    case \"AzureAD\":\r\n        context.userName = \"azad\" + uuid();\r\n        break;\r\n    default:\r\n        log(\"Warning\", \"Provider \" + context.loginProvider + \" was not handled\");\r\n        // Uncomment to generate a username as a uuid\r\n        // context.userName = \"u\" + uuid();\r\n        break;\r\n}\r\n*/\r\n"
+      }
+    },
```

Comment:
Why is this necessary, and is specifying all of it necessary? Please add a comment above it explaining the reason.

Reply:
This is a sample project to represent the capabilities of `Lombiq.Privacy` module.
I think, setting all the site, etc. properties is the best solution to reproduce a clean environment. Who knows what happened with the site before.

Reply:
Okay, but then include a comment explaining this. If I understand correctly all of these are the default settings, except "UsersCanRegister", right? Make sure to indicate which values are customized.